### PR TITLE
add a tool to restore/extract a backup

### LINKED ISSUE #24: add a tool to restore/extract a backup
Extracting a backup from borgbase requires too many manipulations to fetch data that we created. This can be easily helped by an `extract` script.

### PR DESCRIPTION
**Pull request by florentk**
fix #24

### CODE REVIEW

**Review by rgaudin — CHANGES_REQUESTED** [review]
Thank you.

This doesn't work as the name is not exported.

Also, could we get rid of that confirmation message?

```
Remote: Warning: Permanently added the ECDSA host key for IP address '95.217.115.47' to the list of known hosts.
Warning: Attempting to access a previously unknown unencrypted repository!
Do you want to continue? [yN]
```

Otherwise listing and extracting work fine but for extracting we have zero output from borg so it would be better to have some log before/after extracting.

File: bin/restore [hunk 1]
```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+#
+# Author : Florent Kaisser <florent.pro@kaisser.name>
+#
+# ENV :
+# - BW_EMAIL : BitWarden account email used to retrieve the key pair and the BorgBase token
+# - BW_PASSWORD : BitWarden master password
```
[the remark this answers is not in the document]
Comment (florentk) [reply bin/restore:7]: We can remove "master"

**Comment by florentk** [comment]
> This doesn't work as the name is not exported.

ohh, true ! 

> Remote: Warning: Permanently added the ECDSA host key for IP address '95.217.115.47' to the list of known hosts.
> Warning: Attempting to access a previously unknown unencrypted repository!
> Do you want to continue? [yN]
> ```

I look at this.

**Review by rgaudin — APPROVED** [review]
👍

**Pull request merged**